src/olympia/devhub/templates/devhub/new-landing/components/footer.html

```diff
@@ -37,7 +37,7 @@ <h3 class="DevHub-Footer-sections-header">{{ _('Resources') }}</h3>
         <h4>{{ _('Tools') }}</h4>
         <ul>
           <li><a href="{{ url('devhub.validate_addon') }}">{{ _('Add-on Validator') }}</a></li>
-          <li><a href="https://addons.mozilla.org/developers/addon/check-compatibility">{{ _('Compatibility Checker') }}</a></li>
+          <li><a href="addon/check-compatibility">{{ _('Compatibility Checker') }}</a></li>
```

Comment:
@EnTeQuAk  I guess that's because earlier the link was https://compatibility-lookup.services.mozilla.com/ 
I will make the change.